pkg/dashboard/model/alert.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+type Alert struct {
+	ID             int64  `json:"-"  meddler:"id,pk"`
+	Key            string `json:"key"  meddler:"key"`
+	LastSeen       int64  `json:"lastSeen,omitempty"  meddler:"last_seen"`
```

Comment:
This is probably carried over from Kubernetes Events.
It makes sense for Events, but what time information do we need for Alerts?
Thinking of: `fired`, `resolved`